Cargo.toml

```
@@ -1,32 +1,58 @@
 [package]
 name = "nft"
-version = "0.2.10"
-authors = ["Gear Technologies"]
-edition = "2021"
+version.workspace = true
+authors.workspace = true
+edition.workspace = true
+license.workspace = true
 
 [dependencies]
-gstd = { git = "https://github.com/gear-tech/gear.git", features = ["debug"], rev = "5c685d0f15c412ab6ee019ceaf7ce084426dfb68" }
-primitive-types = { version = "0.12.1", default-features = false }
-nft-io = { path = "io" }
-gear-lib = { git = "https://github.com/gear-dapps/gear-lib.git", tag = "0.3.6" }
-gear-lib-derive = { git = "https://github.com/gear-dapps/gear-lib.git", tag = "0.3.6" }
-sp-core-hashing = { version = "8.0.0", default-features = false }
-hashbrown = "0.13"
-gmeta = { git = "https://github.com/gear-tech/gear", rev = "5c685d0f15c412ab6ee019ceaf7ce084426dfb68", features = ["codegen"] }
+gstd.workspace = true
+primitive-types.workspace = true
+nft-io.workspace = true
+gear-lib.workspace = true
+gear-lib-derive.workspace = true
+sp-core-hashing.workspace = true
+hashbrown.workspace = true
+gmeta.workspace = true
 
 [dev-dependencies]
-gtest = { git = "https://github.com/gear-tech/gear.git", rev = "5c685d0f15c412ab6ee019ceaf7ce084426dfb68" }
-hex-literal = "0.4"
-sp-core = { git = "https://github.com/gear-tech/substrate.git", rev = "36699c4478ce8ae05517bb42a1b115403313e083" }
-gclient = { git = "https://github.com/gear-tech/gear.git", rev = "5c685d0f15c412ab6ee019ceaf7ce084426dfb68" }
-tokio = { version = "1", features = ["full"] }
+nft-state = { workspace = true, features = ["binary-vendor"] }
+hex-literal.workspace = true
+sp-core.workspace = true
+gclient.workspace = true
+gtest.workspace = true
+tokio.workspace = true
 
 [build-dependencies]
-gear-wasm-builder = { git = "https://github.com/gear-tech/gear.git", rev = "5c685d0f15c412ab6ee019ceaf7ce084426dfb68" }
-nft-io = { path = "io" }
+nft-io.workspace = true
+gear-wasm-builder.workspace = true
+gmeta.workspace = true
+
+[features]
+binary-vendor = []
+
+[workspace.package]
+version = "0.1.1"
+edition = "2021"
+license = "MIT"
+authors = ["Gear Technologies"]
 
 [workspace]
 members = ["state"]
 
-[features]
-binary-vendor = []
+[workspace.dependencies]
+gstd = { git = "https://github.com/gear-tech/gear.git", features = ["debug"], rev = "78dfa07" }
+gmeta = { git = "https://github.com/gear-tech/gear", rev = "78dfa07", features = ["codegen"] }
```

```suggestion
gmeta = { git = "https://github.com/gear-tech/gear", rev = "78dfa07" }
```
Please enable that only in the `state` crate's `Cargo.toml`.